qiime2/plugins.py

```diff
@@ -490,3 +490,16 @@ def exec_module(self, module):
 
 
 sys.meta_path += [QIIMEArtifactAPIImporter()]
+
+
+def assert_no_nans_in_tables(fh):
+    '''
+    Checks for NaNs present in any of the tables in the indicated file then
+    resets to the head of the file.
+    '''
+    from pandas import read_html
+
+    tables = read_html(fh)
+    for df in tables:
+        assert not df.isnull().values.any()
+    fh.seek(0)
```

Comment:
Wrong plugin, sorry. We want the one which a plugin would import `qiime2.plugin.testing` or something like that.